### PR TITLE
Feature/custom homepage  user profile collections

### PR DESCRIPTION
**Pull request by manuelurenah**
# 🕵️

### CODE REVIEW

File: src/components/Cards/CollectionCard.tsx [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+  const coverImages = getCoverImages();
+
+  return (
+    <FeedCard
+      className={classes.noImage}
+      href={`/collections/${data.id}`}
+      aspectRatio="square"
+      sx={{ position: 'absolute', top: 0, right: 0, bottom: 0, left: 0 }}
+    >
+      <div className={classes.root}>
```
Comment (lrojas94) [inline src/components/Cards/CollectionCard.tsx:45]: Might wanna use the noImage class like in the Model card when Image is null

File: src/server/services/collection.service.ts [hunk 2]
```diff
@@ -47,6 +59,35 @@ export type CollectionContributorPermissionFlags = {
...
 };
 
+export const getAllCollections = async <TSelect extends Prisma.CollectionSelect>({
+  input: { limit, cursor, privacy, types, userId, sort, ids },
+  user,
+  select,
+}: {
+  input: GetAllCollectionsInfiniteSchema;
+  select: TSelect;
+  user?: SessionUser;
```
Comment (lrojas94) [inline src/server/services/collection.service.ts:69]: Doesn't seem used here

**Review by lrojas94 — APPROVED** [review]

File: src/components/Cards/CollectionCard.tsx (hunk 1 above)
Comment (manuelurenah) [reply src/components/Cards/CollectionCard.tsx:45]: It's been used up above in the `FeedCard`

File: src/server/services/collection.service.ts (hunk 2 above)
Comment (manuelurenah) [reply src/server/services/collection.service.ts:69]: For now, but we might it later 🕵️

**Pull request merged**